test/TestRulesCollection.py

```diff
@@ -22,7 +22,6 @@
 import os
 
 import pytest
-
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.